test/test_common/test_base.h

```diff
@@ -23,10 +23,4 @@ class TestListener : public ::testing::EmptyTestEventListener {
   void OnTestEnd(const ::testing::TestInfo& test_info) override;
 };
 
-// TODO(jmarantz): Before Alyssa found this TestListener hook for me, we had
-// merged a PR that added the same functionality via common TestBase classes.
-// These should be removed.
-using TestBase = ::testing::Test;
-template <class T> using TestBaseWithParam = ::testing::TestWithParam<T>;
```

Comment:
This may cause build breakages if folks don't sync to head once this is submitted
Your call if you want to leave it in for a week or two and do one final round of clean up, or tag open PRs which need to sync to head to pick up this change.